File: 0x00-Leap_year/leap.c
#include "leap.h"

/**
 * leap_year - check for leap year
 * @year: to be checked
 * Return: boolean
 */
 
 /* if (year % 4 == 0 && (year % 100 != 0 || year % 400 == 0))
        return (true);
    return (false);

*/


bool leap_year(int year)
{
	return (year % 4 == 0 && (year % 100 != 0 || year % 400 == 0));
}

File: 0x00-Leap_year/leap.py
#!/usr/bin/python3
""" Module contains function leap_year"""


def leap_year(year):
    """ Returns True if the year is leap year; otherwise Returns False

    if ((year % 4 == 0) and (year % 100 != 0 or year % 400 == 0)):
        return True
    return False
    """
    
    return ((year % 4 == 0) and (year % 100 != 0 or year % 400 == 0))
